INPUT/config_modestep.ini

```diff
@@ -0,0 +1,130 @@
+
+[model]
+name = nstx 204202_jul19       ; Label...
+rng_seed = 42           ; Random Number Generator seed
+ntor = 5000             ; not sure what this is
+npert = 2               ; pertubation mode (0 off, 2 enabled)
+; field strength in kgauss, bkg
+; [kG] from LRDFIT06 @ ~450ms
+bkg = 5.72
+
+; numerical equilibrium spdata read
+; can set ripple zero with krip 0
+#krip = 0  ;  comment out for ripple, XXX, change this when find...
+#kripsav = krip
+
+; potential amplitude, keV - from experiment
+pamp  = 2.8
+;  potential profile 1./scale-length, see ptrb2
+rprof = 3.765
+
+; Energy Conservation
+#dele = 5
+
+[input]
+spdata_file = reference_test_case/spdata         ; equilibrium data filename
+#alphas_file = INPUT/alpha_3_9.dat  ; alpha data filename
+displ_file  = reference_test_case/displ_4_5.dat  ; displacements data filename
+fbmdata_file = INPUT/example_fbm_dist.dat ; post processed nubeam output
+
+[output]
+; note initial_update_pdedp may change this to an input...
+pdedp_file = pDEDP.AEP  ; pDEDP file name
+bfield_file = p_bfield.plt      ; bfield plot data file
+; optionally output a sparse compressed file, requires pdedp_sparse_file path
+output_sparse = true
+pdedp_sparse_file = pdedp.nz
+
+
+[pdedp]
+nruns = 10
+compute_pdedp = true
+; The following reads the initial pDEDP data from a file
+; for the first time step, instead of reading
+; from a configuration.
+initial_update_pdedp_from_file = false
+
+; Deposit on bins w/ resonant particles, improves stats
+;   After nruns_pdedp beyond deposit_on_bins_after_fraction
+;   Set to 0 or False to disable.
+deposit_on_bins_after_fraction = 0.5
+
+;Set duration of simulation
+pdedp_dtrun = 0.2 ; [ms]
+
+;Set Micro time step for rescaling DE/dt and DP/dt kicks
+pdedp_dtsamp = 11.854e-3 ; [ms]
+pdedp_dtav = 1.0e-3 ; [ms]
+
+; global - reduce number of samples to manageable size
+pdedp_tskip = 5
+
+; Time step in [ms] to update particle domain info
+pdedp_otpup = 1.e-3
+
+; Flag pdedp_focusdep: when True, will check bin of each
+; particle at deposition. If the bin doesnt contain
+; resonant particles (DE,DP.ne.0) the particle
+; will be thrown away and its E,Pz,mu re-extracted
+; at re-deposition. This increases the statistics
+; around resonances.
+pdedp_focusdep = true
+
+; Optimize (DE,DPz) range during the run when True
+; Optimization is performed on the first run only to avoid
+; excessive interpolations of the pDEDP probabilities.
+; Suggestion: start with a small range for both DE and DPz
+; and let the script expand it based on the actual maximum
+; kicks recorded during the run.
+pdedp_optimize = true
+
+[perturbation]
+; field perturbation choice:
+;  0=none
+;  1=read alphas, compute displacement
+;  2=read displacement, compute alphas
+falf = 13.3550 ; from NOVA analysis
+ascale = 5.
+alimit = 0.
+global_scaling_factor = 3E-4
+freq_scaling_factor = 1.
+sng  = 2.5E-2,
+do_modestep = true
```

Comment:
I expect that additional parameters will be required for the modestep method, for example damping rates for the perturbation. I'll check with Roscoe.